### DISCUSSION TITLE
Comparison with sktime

### DISCUSSION THREAD

**Discussion by SultanOrazbayev** [JoaquinAmatRodrigo/skforecast#660] [General]
Hello, could you kindly point me to a description of the main differences between skforecast and sktime?

**Comment by JoaquinAmatRodrigo** [comment]
Hi @SultanOrazbayev,

**skforecast** and **sktime** are both Python libraries that share some functionality. **skforecast** is primarily focused on providing tools and utilities for time series forecasting using scikit-learn compatible interfaces. It aims to simplify forecasting tasks by providing a familiar API for users already familiar with scikit-learn. **sktime**, on the other hand, is a more comprehensive time series analysis library that covers a broader range of tasks beyond forecasting, such as time series classification, regression, clustering, and more.

In **skforecast** we try to offer a simpler interface compared to **sktime**, with the drawback that some features may be missing.
